frontend/src/pages/index.vue

```diff
@@ -47,8 +46,6 @@ onMounted(() => {
 
 const isXl = computed(() => uiStore.isBreakpoint("xl"))
 
-const isDarkModeSeen = computed(() => uiStore.isDarkModeSeen)
```

Comment:
Thank you for your contribution 😄 The `showThemeSwitcher` needs to be deleted now :)

Reply:
Addressed in 41fdffd47